packages/core/monaco-editor/src/components/MonacoEditor.vue

```diff
@@ -1,16 +1,299 @@
+    --vscode-sash-hoverBorder: var(--kui-color-border-primary, #{$kui-color-border-primary});
+    /* stylelint-enable */
+
+    .sticky-widget {
+      z-index: 2;
+    }
+
+    // Editor's suggestion overlay styles
+    .suggest-details-container {
+      border-radius: var(--kui-border-radius-50, #{$kui-border-radius-50}) !important;
+
+      .suggest-details {
+        border-radius: var(--kui-border-radius-50, #{$kui-border-radius-50}) !important;
+        overflow-x: hidden;
+        overflow-y: auto;
+
+        .type {
+          word-break: break-word !important;
+        }
+      }
+    }
+
+    // Editor's suggestion widget
+    .suggest-widget {
+      border-radius: var(--kui-border-radius-50, #{$kui-border-radius-50}) !important;
+      min-height: 30px !important;
+
+      &.shows-details {
+        border-radius: var(--kui-border-radius-50, #{$kui-border-radius-50}) !important;
+        min-height: 30px !important;
+        overflow-x: hidden;
+        overflow-y: auto;
+        scrollbar-width: thin;
+      }
+
+      .tree,
+      .monaco-scrollable-element,
+      .monaco-list,
+      .monaco-list-rows {
+        border-radius: var(--kui-border-radius-40, #{$kui-border-radius-40}) !important;
+      }
+
+      .monaco-list-rows {
+        overflow-x: hidden;
+        overflow-y: auto;
+      }
+
+      .monaco-list-row {
+        padding: var(--kui-space-0, #{$kui-space-0}) var(--kui-space-30, #{$kui-space-30});
+
+        &.focused {
+          background: var(--kui-color-background-neutral-weaker, #{$kui-color-background-neutral-weaker});
+
+          .monaco-icon-label,
+          .suggest-icon {
+            color: var(--kui-color-text-neutral-strongest, #{$kui-color-text-neutral-strongest}) !important;
+          }
+
+          // The label showing additional details about the suggestion
+          .details-label {
+            color: var(--kui-color-text-neutral-strongest, #{$kui-color-text-neutral-strongest});
+          }
+        }
+
+        // The matching part of the suggestion
+        .highlight {
+          font-weight: var(--kui-font-weight-semibold, #{$kui-font-weight-semibold});
+        }
+
+        &:first-child {
+          &.focused {
+            border-top-left-radius: var(--kui-border-radius-30, #{$kui-border-radius-30});
+          }
+        }
+
+        // The colour of the icons in the suggestion list
+        .suggest-icon {
+          &:not(.codicon-symbol-property) {
+            color: var(--kui-color-text-primary, #{$kui-color-text-primary});
+          }
+        }
+
+        .monaco-icon-label {
+          color: var(--kui-color-text-neutral, #{$kui-color-text-neutral}) !important;
+        }
+      }
+    }
+
+    .codicon-suggest-more-info {
+      transform: translateY(2px) !important;
+    }
+
+    // Editor's search box styles
+    .find-widget {
+      background: var(--kui-color-background, #{$kui-color-background});
```

Comment:
This interpolation syntax is only needed when setting another CSS variable. Here where you're applying directly can use normal syntax 
```suggestion
      background: var(--kui-color-background, $kui-color-background);
```
Same goes for all other instances